vllm/usage/usage_lib.py

```diff
@@ -130,6 +130,7 @@ def __init__(self) -> None:
         self.total_memory: Optional[int] = None
         self.architecture: Optional[str] = None
         self.platform: Optional[str] = None
+        self.cuda_runtime: Optional[str] = None
```

Comment:
also tried to add cuda driver version, but that seems quite complicated. we don't have an easy way to get it. I think cuda runtime version is enough right now.